scripts/exrrfs_radar_refl.sh

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091,SC2153,SC2154
+declare -rx PS4='+ $(basename ${BASH_SOURCE[0]:-${FUNCNAME[0]:-"Unknown"}})[${LINENO}]: '
+set -x
+cpreq=${cpreq:-cpreq}
+
+cd "${DATA}" || exit 1
+
+#
+#-----------------------------------------------------------------------
+#
+# Set environment
+#
+#-----------------------------------------------------------------------
+#
+ulimit -s unlimited
+ulimit -a
+
+#
+#-----------------------------------------------------------------------
+#
+# Extract from CDATE the starting year, month, day, and hour of the
+# forecast.  These are needed below for various operations.
+#
+#-----------------------------------------------------------------------
+#
+YYYYMMDDHH=$(date -d "${CDATE:0:8} ${CDATE:8:2}" +%Y%m%d%H)
+YYYY=${YYYYMMDDHH:0:4}
+MM=${YYYYMMDDHH:4:2}
+DD=${YYYYMMDDHH:6:2}
+HH=${YYYYMMDDHH:8:2}
+YYYYMMDD=${YYYYMMDDHH:0:8}
```

Comment:
CDATE is already in the form of `YYYYMMDDHH`, I think we can use it to replace `YYYYMMDDHH` in lines28-32 directly.

Reply:
replaced